Polyhedron/demo/Polyhedron/triangulate_primitive.h

```diff
@@ -145,6 +150,8 @@ class FacetTriangulator
      previous = vh;
      }
     }
+    if(last_inserted == 0)
```

Comment:
A `Vertex_handle` is not supposed to be comparable to `0`. I may compile, but the correct code is `last_inserted==Vertex_handle()`.